src/common/components/vanilla-modules/forms/form-group-select.js

```diff
@@ -0,0 +1,61 @@
+import React, { PropTypes } from 'react';
+
+import styles from '../../../style/vanilla/css/forms.css';
+
+export default function FormGroupSelect(props) {
```

Comment:
There are small visual bugs with Select, reported to vanilla: https://github.com/vanilla-framework/vanilla-framework/issues/1289